invenio_app/ext.py

```diff
@@ -47,6 +51,27 @@ def init_app(self, app, **kwargs):
             self.talisman = Talisman(
                 app, **app.config.get('APP_DEFAULT_SECURE_HEADERS', {})
             )
+        # Enable PING view
+        if app.config['APP_HEALTH_BLUEPRINT_ENABLED']:
+            try:
+                app.register_blueprint(
+                    import_string(app.config['APP_HEALTH_BLUEPRINT'])()
+                )
+            except ImportError:
+                logging.error('APP_HELATH_BLUEPRINT_ENABLE was set to TRUE but'
```

Comment:
If you just want to enable the default blueprint, won't this cause it to throw the error all the time?